jobs/datadog-cc-worker/templates/process.yaml.erb

```diff
@@ -0,0 +1,6 @@
+init_config:
+
+instances:
+  - name: cc api worker
```

Comment:
Finally we did it with the thresholds in the check. The documentation of the process check says:

https://github.com/DataDog/dd-agent/blob/master/conf.d/process.yaml.example#L25-L31

>    thresholds: (optional) Two ranges: critical and warning
>         warning: (optional) List of two values: If the number of processes found is below the first value or
>                  above the second one, the process check will return WARNING.
>         critical: (optional) List of two values: If the number of processes found is below the first value or
>                   above the second one, the process check will return CRITICAL.
>     In this example, process check will return OK for 3 to 5 process. WARNING for 1, 2, 6, 7 processes and Critical below 1 or above 7.
> CRITICAL is always dominant in case of overlapping.

I do not think that is reflected in the UI of datadog.

/cc @mtekel